docs/README.md

```diff
@@ -175,14 +175,20 @@ Register-OpenTelemetryForIIS
 # Setup your Windows Service instrumentation
 Register-OpenTelemetryForWindowsService -WindowsServiceName "MyServiceName" -OTelServiceName "MyServiceDisplayName"
 
+# Setup environment to start instrumentation from the current PowerShell session
+Register-OpenTelemetryForCurrentSession -OTelServiceName "MyServiceDisplayName"
+
+# Get current installation location
+Get-OpenTelemetryInstallDirectory
+
 # List all available commands
 Get-Command -Module install
 
 # Get command's usage information
 Get-Help Install-OpenTelemetryCore -Detailed
 ```
 
-⚠️ You have to reboot the machine
+⚠️ You have to restart the IIS / Windows service
```

Comment:
Maybe it could add the commands to the snippet as well?

```
Restart-Service -Name "MyServiceName"
& iisreset
```

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1473/commits/fe06b8f6dfbd72d3186e5bb138e7e7a8fbe44bce

Reply:
Warning also had to be changed >  https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1473/commits/9c78a6ff0fec6429f93b555d493771b5a3b42bec